packages/wrangler/src/d1/migrations/apply.tsx

```diff
@@ -90,17 +90,11 @@ export const ApplyHandler = withConfig<BaseSqlExecuteArgs>(
 			return;
 		}
 
-		if (isInteractive() && !CI.isCI()) {
-			const ok = await confirm(
-				`About to apply ${unappliedMigrations.length} migration(s)\n` +
-					"Your database may not be available to serve requests during the migration, continue?",
-				<Box flexDirection="column">
-					<Text>Migrations to be applied:</Text>
-					<Table data={unappliedMigrations} columns={["Name"]}></Table>
-				</Box>
-			);
-			if (!ok) return;
-		}
+		const ok = await confirm(
+			`About to apply ${unappliedMigrations.length} migration(s)
```

Comment:
just realised, we're not printing the table of migrations here any more @penalosa ?